styles.css

```diff
@@ -0,0 +1,231 @@
+@import url('https://fonts.googleapis.com/css2?family=Crete+Round&family=Roboto:wght@700&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Crete+Round&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Poppins&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Inter:wght@600&display=swap');
+
+html {
+  margin: 0;
+  height: 100%;
+  width: 100%;
+}
+
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+  background-color: black;
+  background-image: url('./images/occ-background-1.jpeg');
+}
+
+.logo {
+  position: static;
+  width: 150px;
+  height: 24px;
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: #fff;
+
+  /* Inside Auto Layout */
+
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin: 0.01px 0.01px;
+}
+
+* { box-sizing: border-box; }
+
+a {
+  color: rgb(255, 255, 255);
+  flex-wrap: wrap;
+}
+
+/* header */
+.header {
+  background-color: rgb(66, 63, 63);
+  box-shadow: 1px 1px 4px 0 rgba(0, 0, 0, 0.1);
+  position: fixed;
+  width: 100%;
+  z-index: 3;
+}
```

Comment:
- [ ] Please ensure that the content of your header section is displayed with flexbox